secure_message/services/internal_user_service.py

```diff
@@ -32,10 +32,12 @@ def get_user_details(uuid):
             resp_json = response.json()
         except HTTPError:
             logger.exception("Failed to get user info", uuid=uuid)
-            raise
+            user_details = InternalUserService.get_default_user_details(uuid)
+            return user_details
         except ValueError:
             logger.exception("Failed to decode response JSON", uuid=uuid)
```

Comment:
Might be worth extending the log lines to say its using default. ValueError is interesting

Reply:
I was trying to work it out and Jacob and Adam had approved

Reply:
This is fine for for a quick fix, but realistically this needs looking at again, if a major internal uses leaves and gets deleted our logs could be a mess with exceptions, we should handle it gracefully